applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -217,28 +230,30 @@ int bt_le_audio_tx_send(struct bt_bap_stream **bap_streams, uint8_t *audio_mappi
 	bool ts_common_acquired = false;
 	uint32_t common_interval = 0;
 
-	for (int i = 0; i < streams_to_tx; i++) {
-		if (tx_info_arr[i].iso_tx.seq_num == 0) {
+	for (int i = 0; i < num_tx; i++) {
```

Comment:
Consider adding `tx_info_arr[tx[i].idx.lvl1][tx[i].idx.lvl2][tx[i].idx.lvl3]` as a variable within this loop - the array is accessed this way seven times within the loop, so it'll the save quite a few brackets in the code